inc/admin/class-admin.php

```diff
@@ -146,20 +146,21 @@ public function check_for_event_submissions(){
 				$parent_id = '';
 				$rootCategories = array();
 				$rootTerms = get_terms( array( 'taxonomy' => $taxonomyActive, 'parent' => 0, 'hide_empty' => false ) );
-				foreach ($rootTerms as $rootTerm) {
-					$rootCategories[] = array('id' => $rootTerm->term_id, 'name' => $rootTerm->name);
-				}
-				
+				//use wp_list_pluck for cleaner code.
+				$rootCategories = wp_list_pluck($rootTerms, 'term_id','name');
```

Comment:
[wp_list_pluck](https://developer.wordpress.org/reference/functions/wp_list_pluck/) is a really useful function in WP for this kind of data extarction from Objects or arrays.

Makes for cleaner/simpler  code